eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -5,11 +5,17 @@ param(
   $RepoName,
   # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   $RepoId="$RepoOwner/$RepoName",
+  # CentralRepoId the original PR to generate sync PR. E.g Azure/azure-sdk-tools for eng/common
+  $CentralRepoId,
+  # We start from the sync PRs, use the branch name to get the PR number of central repo. E.g. sync-eng/common-(<branchName>)-(<PrNumber>). Have group name on PR number.
+  $CentralPRRegex,
   [Parameter(Mandatory = $true)]
   $BranchPrefix,  
   # Date format: e.g. Tuesday, April 12, 2022 1:36:02 PM. Allow to use other date format.
   [AllowNull()]
   [DateTime]$LastCommitOlderThan,
+  # Allow to filter by PR creator.
+  $FilterByPRCreator,
```

Comment:
Please filter the PR by creator to avoid deleting unexpected PR/branch which matches the branch prefix.

Reply:
As we discussed offline, adding PR creator check is more safe, but it's also fine if we don't check it. So please delete this parameter.